src/darsia/image/image.py

```diff
@@ -228,7 +228,24 @@ def timestamp(self, time: datetime) -> None:
     def original_dtype(self) -> np.dtype:
         return self.metadata["original_dtype"]
 
-    # ! ---- Corresponding setter functions for metadata
+    # ! ---- Operation overloaders
+    def __sub__(self, other: da.Image):
+        """Subtract two images.
+
+        Arguments:
+            other (Image): image to subtract from self
+
+        Returns:
+            Image: difference image
+        """
+        if self.img.shape != other.img.shape:
+            warn("Images have different shapes. Resizing second argument to match.")
+            other_img = cv2.resize(other.img, self.img.shape[::-1][1:3])
+            return da.Image(self.img-other_img, copy.copy(self.metadata))
```

Comment:
Small typo. Should be `other.img`

Reply:
ah no. The resize is wrong. It should reverse x and y. Or are you trying to do this  here? I am not following the index address. Using tuple(reversed(self.img.shape[:2])) usually worked out for me.

Reply:
I think that the resize is correct. This was just the way I came up with on the fly to reverse x and y. At least it worked when I tested it. Had forgotten the tuple(reversed()) command.

Also I don't think it should be other.img. It is horrible notation from my side though. I'll fix it. And ill ad a copy so that the original other is not modified.

Reply:
Check new update. I changed to the tuple(reversed()), it seems like it does exactly the same, but it reads way nicer, thanks!